### ISSUE TITLE
Error for step Background and foreground colors have a sufficient contrast ratio on mobile report

### ISSUE THREAD

**Issue by GreathostRo** [GoogleChrome/lighthouse#13368]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://beta3.a24.ro/

### What happened?

For several days I have been generating reports for this site and I always receive the same error in the Accessibility step.
Background and foreground colors have a sufficient contrast ratio Error!

### What did you expect?

I want to see errors to optimize website.

### What have you tried?

I generate Lighthouse report for mobile.

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

8.4.0

### Chrome Version

95.0.4638.69 (Official version) (64 bits)

### OS

Windows 11

### Relevant log output

```shell
Runtime Settings
URL
https://beta3.a24.ro/
Fetch Time
Nov 14, 2021, 12:28 AM GMT+2
Device
Emulated Moto G4
Network throttling
562.5 ms HTTP RTT, 1,474.6 Kbps down, 675 Kbps up (DevTools)
CPU throttling
4x slowdown (DevTools)
Channel
devtools
User agent (host)
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36
User agent (network)
Mozilla/5.0 (Linux; Android 7.0; Moto G (4)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4590.2 Mobile Safari/537.36 Chrome-Lighthouse
CPU/Memory Power
1382
Axe version
4.2.3
Generated by Lighthouse 8.4.0 | File an issue
```

**Comment by adamraine** [comment]
This looks like an issue with axe that's fixed in the latest axe-core (4.3.5). We use axe-core 4.2.3 so a version bump is needed here.

Also, DevTools in chrome stable was struggling to finish the Lighthouse run for me. It's working in Canary though.

**Comment by GreathostRo** [comment]
Hi, thanks for the reply, I installed Google Chrome Canary and failed to complete a Lighthouse report. It takes a very, very long time to generate the report and it is not finished. Sometimes it appears in the address bar about:blank and the page changes to white and stays that way. Please check the screenshot attached.

Thank you!

![Captură de ecran 2021-11-26 122411](https://user-images.githubusercontent.com/89145551/143566396-1c88504b-98a7-4058-a8cb-dfefc7fd8c58.png)
